packages/block-editor/src/store/private-selectors.js

```diff
@@ -560,3 +560,23 @@ export function isZoomOutMode( state ) {
 export function getSectionRootClientId( state ) {
 	return state.settings?.[ sectionRootClientIdKey ];
 }
+
+/**
+ * Returns the zoom out state.
+ *
+ * @param {Object} state Global application state.
+ * @return {boolean} The zoom out state.
+ */
+export function getZoomLevel( state ) {
+	return state.zoomLevel;
+}
+
+/**
+ * Returns whether the editor is considered zoomed out.
+ *
+ * @param {Object} state Global application state.
+ * @return {boolean} Whether the editor is zoomed.
+ */
+export function isZoomOut( state ) {
+	return getZoomLevel( state ) < 100;
+}
```

Comment:
Let's make the nomenclature consistent. Is it "zoom-out" or "zoom-level"? `type` and naming need to be consistent.

Reply:
I think this is all right, zoom out is any zoom level less than 1.

Reply:
See https://github.com/WordPress/gutenberg/pull/65482#discussion_r1766606365